src/OneExplorer.ts

```diff
@@ -15,13 +15,34 @@
  */
 
 import * as fs from 'fs';
+import * as ini from 'ini';
 import * as path from 'path';
 import * as vscode from 'vscode';
 
+/**
+ * Read an ini file
+ * @param filePath
+ * @returns `object` if file read is successful, or `null` if file open has failed
+ *
+ */
+function readIni(filePath: string): object|null {
+  let configRaw: string;
+  try {
+    configRaw = fs.readFileSync(filePath, 'utf-8');
+  } catch (e) {
+    console.error(e);
+    return null;
+  }
+
+  // TODO check if toString() is required
+  return ini.parse(configRaw.toString());
+}
```

Comment:
https://github.com/Samsung/ONE-vscode/blob/7eeb7485e08af5642001fb088fbe4c29602f038c/src/Utils/Helpers.ts#L79-L86

It seems that there is a helper which do the same (or pretty similar) usage.

Reply:
Look good. But `loadCfgFile` doesn't handle errors, so may I replace the `readIni` to `loadCfgFile` function later and enhance `loadCfgFile`'s error handling?

Reply:
:+1: